bluesky/log.py

```diff
@@ -106,7 +103,21 @@ def __init__(self, fmt=DEFAULT_FORMAT, datefmt=DEFAULT_DATE_FORMAT,
             self._normal = ''
 
     def format(self, record):
-        record.message = record.getMessage()
+        message = []
+        if hasattr(record, 'old_status'):
```

Comment:
I don't know that we need any special additions to the formatter in this case. It looks like the log messages themselves are enough. I think the value of `extra` will just be in providing structured information to handlers that accept structured data, like logstash.

Reply:
My motivation is let each logging message easy to be figured whether they are create by docuemnt_emit, state_chang or msg command. Put tags in front of message might be useful?

Reply:
I don't think that most users will want to think about it in those terms. A message saying "The state changed from X to Y," or "A __ document has been emitted (uid=__)" is sufficiently clear and nicely readable.

The slightly different approach in caproto was serving a different purpose, where there were many overlapping tags, and a more engineering-focused "audience".